### ISSUE TITLE
[Custom feeds] Land on the pending feed page after creating a second feed 

### ISSUE THREAD

**Issue by mountiny** [Expensify/App#51567]
Reported in Slack [here](https://expensify.slack.com/archives/C06ML6X0W9L/p1729884618757199) - check for more details 

BUG: After adding a second card feed, you're supposed to land on the feed pending screen for the second feed. Instead you land back on the first feed. Shown in :thread:

<details><summary>Issue Owner</summary>Current Issue Owner: @allgandalf</details>

**Comment by koko57** [comment]
I would like to help with this issue 🙂

**Comment by mountiny** [comment]
no payments will be needed

**Comment by koko57** [comment]
Working on this today. I've noticed that we have no loading state like we just applied in the Expensify card when making the call to API - maybe we should add one? And Error handling?

https://github.com/user-attachments/assets/624213ef-c894-47ff-bffe-859fb74f1220

**Comment by koko57** [comment]
PR ready for review. https://github.com/Expensify/App/pull/51719
For now - just landing on a proper page, I can cover the loading button and error handling in another PR

**Comment by lschurr** [comment]
Checked with @mountiny on this one - we will handle payments once the custom feeds project is done. @allgandalf could you work on the checklist? https://github.com/Expensify/App/issues/51567#issuecomment-2462149426

**Comment by mountiny** [comment]
Yes, thanks! once the checklist is done, we can close
